lab1/README.md

```diff
@@ -0,0 +1,104 @@
+# Lab 1 - Restaurant App
+
+Your goal for this lab is to make an app for a restaurant to manage incoming orders from customers.
+
+You will have 1.5 weeks to complete this lab. **Lab 1 is due at 6:30pm on Thursday, October 13th.** This lab will be done in groups of either two or three. There should be two groups of three.
+
+The following are the minimum requirements for this lab. **You must also add at least one additional feature (minimum one, possibly more!) as well.**
+
+## MVP
+
+Your app should have two “users”, cooks and customers.
+
+### Cooks
+
+- There should be a “Menu” with food items such as Hamburgers, Pizza, etc.
+- Each food item should have an associated ingredients list. (ie, Pizza -> dough, sauce, cheese)
+- Cooks can add/edit/delete food items from the Menu.
+- Cooks can add/edit/delete ingredients from each menu item’s ingredients list.
+- Cooks can view “orders”, which each have an associated customer and food item(s).
+- Cooks can mark orders as completed.
+
+### Customers
+
+- Customers can view the menu and make orders by selecting one or more menu items, and the quantities of each item (i.e., 2 burgers, 4 pizzas, etc.).
+- Customers should be able to specify their name when placing an order.
+
+Your app’s data should be persistent on your phone. This means you should store everything in either an S3 or SQL database so that menu items/orders/ingredients are saved when you exit the app.
+
+## Possible Additions (feel free to come up with your own, these are just suggestions)
+
+- Include pictures of menu items (allow a cook to use their phone’s camera to add images of menu items)
+- Customers can customize menu items (Customers can see the ingredients list for menu items, and select certain items to include/not include)
+- Use an external SQL database (so that it syncs across multiple phones)
+- Store your app’s data in [Amazon S3](http://docs.aws.amazon.com/mobile/sdkforandroid/developerguide/s3transferutility.html). This will make it persistent across phones as well!
+- Have a simple login-authentication (prompt cooks/customers for a username, and maybe password. Have a login/add new user feature)
+- Have a complex login-authetication with an API like [facebook](https://developers.facebook.com/docs/facebook-login/android) or [google](https://developers.google.com/api-client-library/javascript/features/authentication).
```

Comment:
I couldn't call it complex (complex kinda sounds like "you have to do more work to login"). Maybe better encrypted login-authentication?